randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -431,6 +450,8 @@ Extra - asset_id: 11
           Any of the following:
               Morph Ball Bombs or Power Bombs ≥ 4
               Power Bombs and Screw Attack
+              # Shinespark across
```

Comment:
This seems wrong. Are you sure that you can do this with **only** one shinespark?
![grafik](https://github.com/randovania/randovania/assets/38186597/07dd3793-ad7e-490a-9dc9-e8b523c3f14c)

Reply:
You're right, this would need at least one PB going forwards and anybombs going backwards. Updated

Reply:
Wait, this already had powerbombs.

Reply:
i didnt see it earlier